### PR TITLE
docs: update Zarf overview to be more clear about how Zarf works

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This updates the Zarf overview to be more clear about what the project is and how to use it.

## Related Issue

Relates to: #369 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Other (security config, docs update, etc)

## Checklist before merging

- [X] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: docs/0-zarf-overview.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ import Tabs from "@theme/Tabs";
...
 
-Zarf is a free and open-source tool that simplifies the setup and deployment of applications and resources onto air-gapped or disconnected environments. Zarf equips you with the ability to quickly and securely deploy modern software onto complex systems without relying on internet connectivity.
+Zarf is a free and open-source tool that simplifies the setup and deployment of applications and supporting resources onto air-gapped or disconnected environments. Zarf equips you with the ability to quickly and securely deploy modern software onto complex systems without relying on internet connectivity.
 
-It also simplifies the installation, updating, and maintenance of DevSecOps capabilities like Kubernetes clusters, logging, and SBOM compliance out of the box. Most importantly, Zarf keeps applications and systems running even when they are disconnected.
+It also simplifies the installation, update, and maintenance of supporting DevSecOps capabilities like Kubernetes clusters and logging stacks, while  providing features like out of the box SBOM compliance. Most importantly though, Zarf keeps applications and systems running even when they are disconnected.
```
Comment (Madeline-UX) [inline docs/0-zarf-overview.md:20]: Reccomendation -  leaning into what we hoped to educate them on and add
"while providing features that ensure supply chain security, including SBOM compliance"

File: docs/0-zarf-overview.md [hunk 2]
```diff
@@ -59,60 +59,66 @@ For more technical information on how Zarf works and to view the Zarf architectu
...
-Zarf enables secure software delivery for various environments, such as remote, constrained, independent, and air-gapped systems. Considering there are various target environments with their own appropriate transferring mechanisms, Zarf does not determine _how_ packages are moved so long as they can arrive in your downstream environment.
+Zarf enables secure software delivery for various environments, such as remote, constrained, standalone, and air-gapped systems. Considering there are various target environments with their own appropriate transferring mechanisms, Zarf does not determine _how_ packages are moved so long as they can arrive in your downstream environment.  See [Package Sources](./4-deploy-a-zarf-package/2-package-sources.md) for more information on where Zarf packages can be stored / pulled from.
 
 ### (3) Deploy the Package
 
 Once your package has arrived, you will need to:
 
 1. Install the binary onto the system.
-2. Run the zarf init package.
+2. Initialize a cluster with a zarf init package.
```
Comment (Madeline-UX) [inline docs/0-zarf-overview.md:98]: may be useful to add the CLI commands here

File: docs/0-zarf-overview.md [hunk 3]
```diff
@@ -59,60 +59,66 @@ For more technical information on how Zarf works and to view the Zarf architectu
...
+### As an Appliance
 
-![Appliance Mode Diagram](.images/what-is-zarf/appliance-mode.png)
+![Appliance Cluster Diagram](.images/what-is-zarf/appliance.drawio.png)
 
-In the simplest usage scenario, your package consists of a single application (plus dependencies) and you configure the Zarf cluster to serve your application directly to end users. This mode of operation is called "Appliance Mode" and it is intended for use in environments where you want to run K8s-native tooling but need to keep a small footprint (i.e. single-purpose/constrained/"Edge" environments).
+In the simplest usage scenario, you deploy the Zarf init package's builtin cluster and use it to serve your application(s) directly to end users. This configuration runs Zarf and it's init package components as a self-contained appliance and is intended for use in environments where you want to run K8s-native tooling but need to keep a small footprint (i.e. single-purpose/constrained/"Edge" environments).
 
-### Utility Cluster Mode
+### With an Existing Cluster
```
Comment (Madeline-UX) [inline docs/0-zarf-overview.md:111]: This change breaks the pattern previously established of using mode name and moving to a description of the type of cluster it creates (but the one above does not follow this new descriptive pattern. I think they should either both be feature names or both describe the type of cluster

**Review by Madeline-UX — COMMENTED** [review]
left some notes

File: docs/0-zarf-overview.md (hunk 2 above)
Comment (Racer159) [reply docs/0-zarf-overview.md]: done

File: docs/0-zarf-overview.md (hunk 1 above)
Comment (Racer159) [reply docs/0-zarf-overview.md]: done

File: docs/0-zarf-overview.md (hunk 3 above)
Comment (Racer159) [reply docs/0-zarf-overview.md]: Is this better now?

**Review by Noxsios — APPROVED** [review]

File: docs/0-zarf-overview.md (hunk 3 above)
Comment (Madeline-UX) [reply docs/0-zarf-overview.md]: Yes

**Pull request merged**